fix(Playwright): Garantindo instalação do firefox no playwright ao executar um scraper

CDGodoy opened the pull request:
<!--
Use título no formato "<tipo>(<escopo>): resumo curto", conforme Política de Commits.
-->

# Descrição
Alterado o makefile incluindo novos procedimentos para que seja instalado o firefox antes de executar um scraper, para garantir que o mesmo esteja instalado.
Usado para solucionar a issue https://github.com/EH-FAKE/check-up/issues/48

## 🗂️ Tipo de Mudança
- [X] 🐞 Correção de bug (*fix*)
- [ ] ✨ Nova funcionalidade (*feat*)
- [ ] ♻️ Refatoração (*refactor/improve*)
- [ ] 📝 Documentação (*docs*)
- [ ] 🚀 Performance (*perf*)
- [ ] 🧪 Testes (*test*)
- [ ] ⚙️ Build/CI (*build/ci/cd/static*)

## 📌 Checklist
- [X] Mensagem de commit segue o padrão **Conventional Commits**  
- [X] Nome da branch segue a **Política de Branches Git Flow**  
- [ ] Testes adicionados/atualizados  
- [ ] Documentação atualizada (README / Storybook / Swagger / MkDocs)  
- [ ] Pipeline CI/CD aprovado  
- [ ] Sem dependências pendentes

## 🔗 Issues Relacionadas
Closes #48 

## 🧪 Como Testar
1. Passo: Execute a configuração inicial do projeto
3. Passo: Execute um Crawler e um Scraper
4. Resultado esperado: O Scraper funcionará devidamente

## 📷 Evidências
Screenshots, GIFs ou logs (se aplicável).

## 📃 Notas Adicionais
Observações para revisores ou deploy.

luciano-freitas-melo committed:
docs(guia-contribuicao): cria guia de contribuicao

luciano-freitas-melo committed:
docs(remove-documetacao): remove readme de issues

luciano-freitas-melo committed:
docs(codigo-conduta): cria codigo de conduta

luciano-freitas-melo committed:
docs(readme): altera parte de contribuicao

CDGodoy committed:
Adiciona correção que instala o firefox ao iniciar qualquer scraper.

CDGodoy committed:
fix: Ajustando instalação do firefox no playwrigth

Co-Authored-By: alanagabriele

luciano-freitas-melo commented on Makefile:
```diff
@@ -62,6 +62,11 @@ scrape_no_openai:
+# Comando para verificação de instalação do Playwright
+test_playwright:
+	@echo "Verificando instalação do Playwright..."
+	docker compose exec scraper python -c "from playwright.sync_api import sync_playwright; print('Iniciando teste do Playwright'); p = sync_playwright().start(); print('Playwright iniciado com sucesso'); browser = p.firefox.launch(); print('Navegador lançado com sucesso'); page = browser.new_page(); print('Nova página criada'); page.goto('https://www.example.com'); print('Página carregada'); browse ...
```
Utilizar `run` é mais recomendado para testes pois não mexe com o serviço principal, criando um container a parte temporario para testar o playwright.

```suggestion
	docker compose run --rm scraper python -c "from playwright.sync_api import sync_playwright; print('Iniciando teste do Playwright'); p = sync_playwright().start(); print('Playwright iniciado com sucesso'); browser = p.firefox.launch(); print('Navegador lançado com sucesso'); page = browser.new_page(); print('Nova página criada'); page.goto('https://www.example.com'); print('Página carregada'); browser.close(); p.stop(); print('Teste do Playwright concluído com sucesso')"
```

luciano-freitas-melo reviewed (approved):
Muito bom! Fiz somente uma sugestão para executar o script do de teste do playwright.

DanielRogs committed:
feat(config): atualiza links de contato no template de issues (#15)

* fix: corrige links e atualiza config de issues do repositorio

* fix(link documentacao):  atualiza link doc novo spider

---------

Co-authored-by: luciano-freitas-melo <lucianodefreitasmelo@hotmail.com>

mateus9levy committed:
feat/ adicionar scraper e spider jornal da paraiba

vinialves2020 committed:
Feat(FrontEnd) adição de icones para os respectivos portais (#27)

* adição de icones para os respectivos portais

* refactor(frontend/Home): remove comentarios de debug


---------

Co-authored-by: luciano-freitas-melo <lucianodefreitasmelo@hotmail.com>

alanagabriele committed:
Merge pull request #2 from GCES-EhFake-Fork/docs/guia-contribuicao

docs(nova documentacao): criação guia de contribuição e codigo de conduta

alanagabriele committed:
Merge pull request #29 from GCES-EhFake-Fork/feat/jornal_da_paraiba

feat(Jornal da Paraiba): Criação do spider e scraper do portal de notícias Jornal da Paraiba

luciano-freitas-melo committed:
build(Makefile): melhoria na execucao dos containers scraper

luciano-freitas-melo committed:
build(dockerfile): ajuste para merge no fork

- PR https://github.com/GCES-EhFake-Fork/checkUp/pull/29/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557 foi mergeado antes e deu conflito no dockerfile
- foi mantido as alterações da branch fix/playwright-browser

luciano-freitas-melo committed:
Revert "build(dockerfile): ajuste para merge no fork"

This reverts commit 95ae13a8a2366e5e353cc81aea1567acb2dca364, reversing
changes made to d73c81bc96271a739e2cfac988c0f5934e1b26ce.
